docs/course_materials/functions.html

```diff
@@ -167,26 +167,25 @@ <h3><b>Example - is even? </b></h3>
                 <section data-background-size="cover" data-background-image="media/exr_background.jpeg" data-markdown="exercises/06_Functions/class_exercise_2.md" data-separator="^\n\n\n" data-separator-vertical="^\n\n">
                 </section>
                 <section>
-                  <h3><b>Example - is prime? </b></h3>
+                  <h3><b>Example - Recursion</b></h3>
                   <ul>
-                    <li>A prime number is a natural number greater than 1 that is not a product of two smaller natural numbers.</li>
+                    <li>A function that calls itself until a specified condition is met.</li>
                   </ul>
                   <pre><code data-trim data-noescape class="hljs lang-python">
-def is_prime(x):
-    if x < 2:
-        return False
-    for n in range(2, x - 1):
-        if x % n == 0:
-            return False
-    return True
+def my_print(message, repeat_times):
+    if repeat_times == 0:
+        return
```

Comment:
note: the students might not be familiar with an empty 'return'.
need to be sure they are familiar with 'end term' as in while loops

Reply:
Done